hawkular-bus-common/src/main/java/org/hawkular/bus/common/consumer/RPCBasicMessageListener.java

```diff
@@ -79,7 +79,7 @@ protected void setMessageSender(MessageProcessor messageSender) {
     }
 
     @Override
-    public void onMessage(Message message) {
+    public void onMessage(javax.jms.Message message) {
```

Comment:
Because at some point when I was introducing the interface for the class that previously used to be called `BasicMessage` I called that new interface simply `Message` and that resulted in import conflicts that I first tried to resolve through fqing javax.jms.Message but then I have given up and used the `BasicMessage` name for the interface renaming the original `BasicMessage` to something else.

Let me unfq back the uses of `javax.jms.Message`